roles/servicetelemetry/templates/manifest_grafana.j2

```diff
@@ -40,13 +37,12 @@ spec:
     - -provider=openshift
     - -pass-basic-auth=false
     - -https-address=:3002
-    - -htpasswd-file=/etc/proxy/htpasswd/auth
     - -tls-cert=/etc/tls/private/tls.crt
     - -tls-key=/etc/tls/private/tls.key
     - -upstream=http://localhost:3000
     - -cookie-secret-file=/etc/proxy/secrets/session_secret
     - -openshift-service-account=grafana-serviceaccount
-    - '-openshift-sar={"resource": "namespaces", "verb": "get"}'
+    - '-openshift-sar={"namespace":"{{ ansible_operator_meta.namespace }}","resource": "grafana", "group":"integreatly.org", "verb":"get"}'
```

Comment:
Another reminder for myself that this sort of change is release-notes worthy.  Basically I'm proposing that instead of requiring accounts with cluster-wide admin-like permissions (the ability to see all namespaces) we allow any accounts with access to read grafana objects in our namespace.

Reply:
FWIW, making this SAR the same as the one we use on prometheus (permission to read prometheus objects in our namespace) would also make sense to me.

Reply:
I had to read through https://github.com/openshift/oauth-proxy#limiting-access-to-users to get a better understanding about what this was doing, but now I get it :)